_data/tagging-status.yml

```diff
@@ -4258,6 +4297,16 @@
    tests: false
    updated: 2024-07-21
 
+ - name: halloweenmath
+   type: package
+   status: unknown
+   included-in: [arxiv001]
```

Comment:
that's funny that this shows up in arxiv001 but I believe you

Reply:
I thought so too :)